lib/elixometer.ex

```diff
@@ -348,8 +348,7 @@ defmodule Elixometer do
       subscribe_options = cfg[:subscribe_options] || []
 
       if reporter do
-        :exometer.info(metric_name)
-        |> Keyword.get(:datapoints)
+        :exometer.info(metric_name, :datapoints)
```

Comment:
Not sure, I'm not very good at reading Erlang tbh, but if I `rebar3 shell` into the latest `exometer_core` dep, I get:
```
4> exometer:new([test], counter).                
ok
5> exometer:info([test]).
[{name,[test]},
 {type,counter},
 {behaviour,undefined},
 {module,exometer},
 {status,enabled},
 {cache,0},
 {value,[{value,0},{ms_since_reset,10118}]},
 {timestamp,221873694821},
 {options,[]},
 {ref,undefined}]
6> exometer:info([test], datapoints).
[value,ms_since_reset]
```
I'm sorry I don't have the type error that led me to debugging this, but it was essentially saying Keyword.get wasn't passing an enumerable because it was nil.